src/run.ts

```diff
@@ -278,22 +284,33 @@ ${
 
   let searchResult = await searchResultPromise;
   console.log(JSON.stringify(searchResult.data, null, 2));
+
+  let messageBody = await prBodyPromise;
+  if (messageBody.length > MAX_CHARACTERS_PER_MESSAGE) {
+    console.log(
+      `message body too long, truncating at ${MAX_CHARACTERS_PER_MESSAGE} charachters.`
+    );
+    messageBody = [
+      messageBody.substring(0, MAX_CHARACTERS_PER_MESSAGE),
+      "[Too long, message truncated]",
+    ].join("\n\n");
```

Comment:
This has the potential of breaking some formatting stuff or breaking links etc (which has probably happened on the attached screenshot). I wonder if on such an occasion, we shouldn't just replace the whole "changelogs" part with a placeholder message explaining the problem and advising to read the git diff, we could even construct a link to the "File Changed" tab there. WDYT?

Reply:
Yes, formatting might be broken. I don't have a strong opinion to be honest.
The important part is that the PR is opened without errors, as otherwise the release is blocked.

Up to you to decide.

Reply:
I think it's safer to start with the suggested way.

We'd have to just move this part to compute a separate object:
https://github.com/changesets/action/blob/7e613b338ee4b15ca9b286bbd31bd1dd31e75594/src/run.ts#L245

and construct the final body later with the algorithm like:
1. prepare the message without considering the limit
2. if the limits has been exceeded then construct a fallback body that would have those things in it:
- ["this prefece"](https://github.com/changesets/action/blob/7e613b338ee4b15ca9b286bbd31bd1dd31e75594/src/run.ts#L226)
- information that the limits has been exceeded
- list of releases (names + versions, similar to what we already print in the full body, just without the changelog part

Reply:
@Andarist I pushed some changes following your suggestions. It would be nice if you can take a look when you have time.
Thanks